packages/api/src/Api.ts

```diff
@@ -25,7 +25,7 @@ import { getProvider } from './util/getProvider';
 import { getTimeout } from './util/getTimeout';
 
 export class Api extends ApiPromise {
-  static create(options: ApiOptions = {}): Promise<Api> {
+  static async create(options: ApiOptions = {}): Promise<Api> {
```

Comment:
not sure if this has to be async..
async and await are used together and 
when a function returns a new Promise it need not be async..

Reply:
there were 2 conflicting lints. one requires `async` to use an `await` inside, the other requires `async` when function returns a promise